edith-step/src/App.js

```diff
@@ -1,19 +1,46 @@
 import React, {Component} from 'react';
 import ReceiptInput from './ReceiptInput';
-// import "./App.css";
+import LineChart, {BarChart} from './UserChart';
+import axios from 'axios';
+//import "./App.css";
 
 
 /** Main webpage for the website. */
 class App extends Component {
-  /** Renders the main webpage. */
+  constructor() {
+    super();
+    this.state = { "chartType" : LineChart }; 
+    this.updateChartType = this.updateChartType.bind(this);   
+  }
+
+  updateChartType(event) {
+    switch(event.target.value) {
+      case "LineChart": 
+        this.setState({"chartType": LineChart});
+        break;
+      case "BarChart":
+        this.setState({"chartType": BarChart});
+        break;
+    }
+  }
+
   render() {
-    /** Returns main components. */  
+    const Chart = this.state.chartType;
     return (
-      <div className='App'>
+      
+      <div className="App">
+        <div onChange={this.updateChartType}>
+          <input defaultChecked type="radio" value="LineChart" name="gender" /> Line
```

Comment:
Should this be a form instead ?

Reply:
It could be, I chose to not make it a form since an HTTP request isn't being made.

Reply:
Oh, I got you !